wurst/util/GroupUtils.wurst

```diff
@@ -97,6 +100,56 @@ function createGroups(int number)
 			+ "All newly created groups will be non-recyclable.")
 		shownMaxError = true
 
+/** used to get units in rect with filter ! caution group needs to be released using group.release() after usage !*/
```

Comment:
GroupUtils is more about utilizing groups correctly than having utility functions for getting them, so these should be in another package, I guess.

Reply:
Also, I think these functions are bad for standart, because they implicitly create groups, *even if you state this in hotdoc*. And IMHO they are easily replicated on the spot, providing much clearer code.

Reply:
true, i already supposed to remove all fitler stuff from the PR